code/modules/flufftext/Hallucination.dm

```diff
@@ -1294,3 +1295,29 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 	H.preparePixelProjectile(target, start)
 	H.fire()
 	qdel(src)
+
+/datum/hallucination/sleeping_carp
+
+/datum/hallucination/sleeping_carp/New(mob/living/carbon/C, forced = TRUE)
+	set waitfor = FALSE
+	..()
+	var/list/mobsyup = list()
+	for (var/mob/living/carbon/A in orange(C,2))
```

Comment:
Why two tiles orange for a thing that should only happen at melee range? Also edge cases for people behind walls.

Reply:
Because I wasn't sure if orange(C,1) would only get the the things on the same tile, but also you can see in the next line I make sure it's in melee range.